include/aws/http/connection_manager.h

```diff
@@ -25,6 +25,23 @@ typedef void(aws_http_connection_manager_on_connection_setup_fn)(
 
 typedef void(aws_http_connection_manager_shutdown_complete_fn)(void *user_data);
 
+/**
+ * Metrics for logging and debugging purpose.
+ */
+struct aws_http_manager_metric {
```

Comment:
trivial/debatable: sounds weird to me as "metric". "metrics" sounds more natural.